From cd59e670a8aa9aeb28d3dfa92cb79b460aa98cb1 Mon Sep 17 00:00:00 2001
From: "David.Wu" <david.wu@rock-chips.com>
Date: Fri, 7 Dec 2018 14:53:10 +0800
Subject: [PATCH] arm64: dts: rockchip: Change the dr_mode from peripheral to
 otg for rk1808

Use the driver to auto switch the host/peripheral mode for rk1808,
so the dr_mode must be otg.

Change-Id: I9b05e06bacd141d5fbd00a9751f2a12a4e4385c8
Signed-off-by: David.Wu <david.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 52ac30796ced..c6501b8d28c9 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -231,7 +231,7 @@
 			compatible = "snps,dwc3";
 			reg = <0x0 0xfd000000 0x0 0x200000>;
 			interrupts = <GIC_SPI 75 IRQ_TYPE_LEVEL_HIGH>;
-			dr_mode = "peripheral";
+			dr_mode = "otg";
 			phys = <&u2phy_otg>, <&combphy PHY_TYPE_USB3>;
 			phy-names = "usb2-phy", "usb3-phy";
 			phy_type = "utmi_wide";
-- 
2.35.3

